app/Http/Controllers/Teams/MembersController.php

```diff
@@ -9,7 +9,6 @@
 
 use App\Http\Controllers\Controller;
 use App\Models\Team;
-use App\Models\TeamMember;
```

Comment:
import still being used 👀

Reply:
also leaving team should also remove from channel?

Reply:
oh yeah forgot about that after merging 🤔